requirements.txt

```
@@ -2,4 +2,7 @@ dask>=2021.02.0
 distributed>=2021.02.0
 kubernetes>=12.0.1
 kubernetes-asyncio>=12.0.1
-kopf>=1.35.3
\ No newline at end of file
+kopf>=1.35.3
+bokeh>=2.4.3
+ipywidgets>=7.7.1
+numpy==1.23.0
```

I don't think we should be hard pinning this here. Probably needs to be a greater than.

```suggestion
numpy>=1.23.0
```

Typically installing `dask[complete]` with `pip` or `dask` with `conda` brings in these dependencies so I'm a little hesitant to add them here too.

---

I've moved them to `requirements-test.txt` but let me know if we want to remove them altogether since they come along with `dask`.

---

Yeah we could switch the dask dependency in `requirements-test.txt` to `dask[complete]` which should pull those in. That means we don't need to maintain the minimum versions in multiple places.

---

Added `dask[complete]` to `requirements-test.txt`.